### DISCUSSION TITLE
VERCEL_GIT_COMMIT_SHA not available in Sveltekit browser

### DISCUSSION THREAD

**Discussion by erezarnon** [vercel/community#2226] [Help]
### Summary

I'm trying to get access to the VERCEL_GIT_COMMIT_SHA in the browswer in sveltekit. The [docs](https://vercel.com/docs/concepts/projects/environment-variables/system-environment-variables) list a SvelteKit legacy beta section which I tried to follow, but build is failing saying the variable doesn't exist.

I tried PUBLIC_VERCEL_GIT_COMMIT_SHA and VERCEL_GIT_COMMIT_SHA. The former I tried to load from both $env/static/public and $env/dynamic/public.

Anyone know how to get access to the sha1 in the browser?

**Comment by amyegan** [comment]
Hi @erezarnon! Access to environment variables in the browser usually requires a prefix which signals that it's safe for the variable to be expose in client-side code. The default [publicPrefix](https://kit.svelte.dev/docs/configuration#env) for SvelteKit is `PUBLIC_` now, but for legacy apps and some other configurations the prefix `VITE_` should be used.

**Comment by erezarnon** [comment]
That variable doesn't have any prefix when I run `vercel env pull`. How do I make it public?

**Comment by erezarnon** [comment]
@amyegan should this variable have a prefix when I run `vercel env pull`?
